alkira/resource_alkira_connector_ipsec.go

```diff
@@ -58,9 +58,10 @@ func resourceAlkiraConnectorIPSec() *schema.Resource {
 							Optional: true,
 						},
 						"enable_tunnel_redundancy": {
-							Description: "indicates to Alkira that health for the site should be reported as good as long as any one of the tunnels is up.",
-							Type:        schema.TypeBool,
-							Optional:    true,
+							Description: "indicates to Alkira that health for the site should " +
```

Comment:
indicates should be capitalized

Reply:
Yeah, good point indeed, let me find if any better wording for this...